### ISSUE TITLE
target_temperature unit not working.

### ISSUE THREAD

**Issue by skavan** [artem-sedykh/mini-climate-card#151]
Hi,
First of all - great card. I'm a bit of pixel-perfect guy so spent 280 lines of yaml to fine tune it!
Here's what it looks like:
<img width="274" alt="image" src="https://github.com/artem-sedykh/mini-climate-card/assets/2557753/7b2f10c6-ee0a-4f5f-b513-57d84e396004">
The one thing I can't figure out how to get 67°C to read 67°F.
I thought it was:
```
- type: custom:mini-climate
  entity: climate.porch_ac_windfree_3_0e
  icon: blank
  scale: 1
  hide_current_temperature: true
  target_temperature:
    icons:
      up: mdi:chevron-up
      down: mdi:chevron-down
    unit: '°F'
```
But it doesn't work. Am I missing a trick, or is it a bug?

p.s. Here's the off display (a few posts have asked about hiding stuff, and greying out stuff -- card_mod is your friend!).
If anyone cares I can post all 280 lines of yaml!
<img width="275" alt="image" src="https://github.com/artem-sedykh/mini-climate-card/assets/2557753/f42610cc-06de-4636-8d2b-4043fa134c8f">

**Comment by regevbr** [comment]
You need to set the unit also on `temperature` and not just `target_temperature`

**Comment by skavan** [comment]
Success. Thank you. You might want to note that in the readme unless I missed it. It's not entirely logical! Very cool card.
When I have time, I'll write my huge card_mod block; perhaps you would consider putting some of the functionality in the core.

**Comment by regevbr** [comment]
![image](https://github.com/artem-sedykh/mini-climate-card/assets/5775519/06207a97-1f36-45b3-ad57-b086cf5f7474)
If you feel it requires a change a PR is more than welcome

**Comment by regevbr** [comment]
Actually it is confusing, I see what is the issue in the code I will fix

**Comment by regevbr** [comment]
released in 2.7.2 @skavan can you please verify it is fixed by removing the unit in the temperature entry?

**Comment by skavan** [comment]
Looks like it's fixed!
